.github/workflows/build-latest.yaml

```diff
@@ -0,0 +1,43 @@
+---
+  name: Build Latest
+  
+  on:
+    workflow_dispatch:  # This allows to kick-off the action manually from GitHub Actions
+    push:
+      branches: [ main ]  # This watches pushes to the "main" branch
+  
+  env:
+    UUID: ${{ github.run_id }}
+  
+  jobs:
+    build:
+      runs-on: ubuntu-latest
+      permissions:
+        contents: read
+        packages: write
+      steps:
+        - name: Set up QEMU
+          uses: docker/setup-qemu-action@v2
```

Comment:
Can you say more about what this is for?

Reply:
Builds a package (docker images) for Kubernetes … https://github.com/mlibrary/dor-py/pkgs/container/dor-py%2Fdor-py

Reply:
Hm, I understand generally the purpose of the GitHub Action, I was asking specifically about QEMU, since I don't think I've seen this in other similar build actions.

Reply:
Ah, ’tis an optimization to cache images for actions.  Probably overkill, but again, just cutting and pasting from other projects.